src/mbgl/renderer/layers/render_line_layer.cpp

```diff
@@ -485,50 +485,92 @@ void RenderLineLayer::update(gfx::ShaderRegistry& shaders,
 
         // interpolation UBOs
         const float zoom = static_cast<float>(state.getZoom());
```

Comment:
`zoom` could change every frame, therefore the interpolation UBOs could also be different for data driven properties. Not updating these after the first compute could be affecting visuals?

Reply:
This code is running per frame, so the zoom is updated per frame.

Reply:
correct, my bad.